crates/loader/src/local/mod.rs

```diff
@@ -40,12 +41,13 @@ pub async fn from_file(
     app: impl AsRef<Path>,
     base_dst: impl AsRef<Path>,
     bindle_connection: &Option<BindleConnectionInfo>,
+    components: Vec<OsString>,
```

Comment:
`components` is a bit undescriptive here - aren't the "components" in the app file?  I realise it's more verbose but maybe a name like `include_components` or `include_only` would give the reader a clearer idea of what the parameter does?